### DISCUSSION TITLE
Line break in sticky header

### DISCUSSION THREAD

**Discussion by ghost** [templaza/astroid-framework#641] [Q&A]
I'm learning about the Astroid Framework on a local development environment. I have a question about the sticky header. In my main menu all menu items are on one line, but in the sticky header there is a line break and a menu item is on a new line.
After looking in the inspector, I have now entered the following into my custum.css.

`*, *::before, *::after {
  box-sizing: unset;
}`

This means that all menu entries are on one line in the sticky header, but is that the right way?

Maybe someone can tell me something about this?

Many thanks in advance

**Comment by Chacapamac** [comment]
Look at the sticky here, this is how it work. Tell me if you see something weird about it.
—> https://grafcomm.ca/

**Comment by ghost** [comment]
I know how sticky headers work, but I had a line break in the sticky header menu that was not present in the main menu. Just like in this example.

Main Menu:

item item item item item item Item

Sticky Header Menu:

Item Item Item Item Item Item Item
Item

I hadn't seen that there are extra settings for the sticky header menu in the Astroid Manager. That was my mistake.

They have done this very nicely on their website with their logo. I think it's nice that it overlays the slideshow. How did you do that? With z-index?

**Comment by Chacapamac** [comment]
For the floating logo I think the best and flexible way to do that is that way.....

**• Add a Joomla Module to the Header Section**
![Astroid-Logo-Module](https://github.com/templaza/astroid-framework/assets/1907276/02759b55-795b-411c-a2a3-b0d3cf257a44)

**• Create the corresponding Joomla Module** _(Great if logo is different in different language)_
![Joomla logo-module](https://github.com/templaza/astroid-framework/assets/1907276/9432ea46-48db-4468-ad38-c0ae274d635f)

• After that it’s all CSS of the logo container and the image itself.
— You can also mix or even recreate (I think) with the Logo Images directly In Astroid Administration
_Here the float code:_
![float-code](https://github.com/templaza/astroid-framework/assets/1907276/2c8e2512-d617-4cc3-89ef-2001aefe512a)

**Another website with this setup.** —>  https://efoilsurf.ca
